### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by annaswims**
## Description of change
Making updates to documentation

### CODE REVIEW

File: app/swagger/readme.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Display the Swagger Doc
 - The doc is served from the endpoint: `GET` `v0/apidocs`
+- [Swaggeer docs](https://department-of-veterans-affairs.github.io/va-digital-services-platform-docs/api-reference/#/) are published or can be generated locally.
```
Comment (omgitsbillryan) [inline app/swagger/readme.md]: Neato. This belongs in it's own section at the top of the readme IMO. Maybe like:

```
# View Swagger Deployed to DEV
[We host the SwaggerUI here](https://department-of-veterans-affairs.github.io/va-digital-services-platform-docs/api-reference/#/), which points itself at our `http://dev-api.va.gov/v0/apidocs` endpoint.

# Display the Swagger Doc Yourself
...blah...
...blah...
```

File: docs/setup/native.md [hunk 2]
```diff
@@ -63,14 +63,18 @@ the `vets-api` directory, run `./bin/setup-osx && source ~/.bash_profile && cd
...
-   - Copy the [certificate][certificate] to `./config/certs/vetsgov-localhost.crt`
-   - Copy the [key][key] to `./config/certs/vetsgov-localhost.key`
-   - *NOTE*: If you don't have access to these keys, running the following
-     commands will provide basic functionality, such as for running unit tests:
-   - `touch ./config/certs/vetsgov-localhost.crt`
-   - `touch ./config/certs/vetsgov-localhost.key`
+   - *NOTE*: If you don't have access to these keys, [Create your own self-signed certificate](https://github.com/department-of-veterans-affairs/vets.gov-team/tree/master/Products/Identity/Login/IDme/development-certificates#creating-your-own-self-signed-certificates)
+   - Copy the [certificate](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Identity/Login/IDme/development-certificates/vetsgov-localhost.crt) to `./config/certs/vetsgov-localhost.crt`
+   - Copy the [key](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Identity/Login/IDme/development-certificates/vetsgov-localhost.key) to `./config/certs/vetsgov-localhost.key`
+
```
Comment (omgitsbillryan) [inline docs/setup/native.md]: All of these links point to the `vets.gov-team` repo, which is private. The problem is that `vets-api` rails server wont even start unless these files exist. So for peeps w/o access to `vets.gov-team`, it's important for them to understand that empty files will at least allow them to startup the server.

File: app/swagger/readme.md [hunk 3]
```diff
@@ -1,7 +1,9 @@
 # Display the Swagger Doc
 - The doc is served from the endpoint: `GET` `v0/apidocs`
 
-# Run the Swagger UI Container
+# View Swagger Deployed to DEV
+- [We host the SwaggerUI here](https://department-of-veterans-affairs.github.io/va-digital-services-platform-docs/api-reference/#/)
```
Comment (johnpaulashenfelter) [inline app/swagger/readme.md]: I *literally* just looked this up through searching GitHub. Thanks!

File: docs/setup/native.md [hunk 4]
```diff
@@ -47,8 +47,8 @@ the `vets-api` directory, run `./bin/setup-osx && source ~/.bash_profile && cd
 1. Install Poppler
    -  `brew install poppler`
 1. Install ClamAV
-  - `brew install clamav`
-  - Take note of the the post-install instructions `To finish installation & run
+   - `brew install clamav`
```
Comment (johnpaulashenfelter) [inline docs/setup/native.md]: It looks like extra spaces here? Linter won't catch MD right now I don't think

File: docs/setup/native.md (hunk 2 above)
Comment (johnpaulashenfelter) [reply docs/setup/native.md]: Plus they're either moving to `va.gov-team` or the sensitive repo.

**Pull request merged**